<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="TC001"
           tests="2"
           failures="2"
           errors="0"
           skipped="0"
           time="2.038">
  <properties />
            <testcase name="test1" time="0.006" classname="com.easy.automation.testcase.TC001">
                <failure type="java.lang.AssertionError"
                  message="expected [8] but found [7]"
              >
<![CDATA[java.lang.AssertionError: expected [8] but found [7]
  org.testng.Assert.fail(Assert.java:94)
  org.testng.Assert.failNotEquals(Assert.java:513)
  org.testng.Assert.assertEqualsImpl(Assert.java:135)
  org.testng.Assert.assertEquals(Assert.java:116)
  org.testng.Assert.assertEquals(Assert.java:190)
  org.testng.Assert.assertEquals(Assert.java:200)
  com.easy.automation.testcase.TC001.test1(TC001.java:15)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1137)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
  org.testng.TestRunner.privateRun(TestRunner.java:753)
  org.testng.TestRunner.run(TestRunner.java:607)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:368)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:363)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:321)
  org.testng.SuiteRunner.run(SuiteRunner.java:270)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1284)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1209)
  org.testng.TestNG.runSuites(TestNG.java:1124)
  org.testng.TestNG.run(TestNG.java:1096)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:282)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:83)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:114)
  org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:286)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:240)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
      </failure>

          </testcase>
            <testcase name="test2" time="2.032" classname="com.easy.automation.testcase.TC001">
                <failure type="org.openqa.selenium.WebDriverException"
                  message="chrome not reachable
  (Session info: chrome=64.0.3282.186)
  (Driver info: chromedriver=2.36.540470 (e522d04694c7ebea4ba8821272dbef4f9b818c91),platform=Windows NT 10.0.16299 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 0 milliseconds
Build info: version: &apos;3.11.0&apos;, revision: &apos;e59cfb3&apos;, time: &apos;2018-03-11T20:26:55.152Z&apos;
System info: host: &apos;DESKTOP-IGA56E0&apos;, ip: &apos;192.168.100.233&apos;, os.name: &apos;Windows 10&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;1.8.0_91&apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, acceptSslCerts: false, applicationCacheEnabled: false, browserConnectionEnabled: false, browserName: chrome, chrome: {chromedriverVersion: 2.36.540470 (e522d04694c7eb..., userDataDir: C:\Users\HePing\AppData\Loc...}, cssSelectorsEnabled: true, databaseEnabled: false, handlesAlerts: true, hasTouchScreen: false, javascriptEnabled: true, locationContextEnabled: true, mobileEmulationEnabled: false, nativeEvents: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: XP, platformName: XP, rotatable: false, setWindowRect: true, takesHeapSnapshot: true, takesScreenshot: true, unexpectedAlertBehaviour: , unhandledPromptBehavior: , version: 64.0.3282.186, webStorageEnabled: true}
Session ID: 7f077e432760af637bbc29581bc42eba"
              >
<![CDATA[org.openqa.selenium.WebDriverException: chrome not reachable
  (Session info: chrome=64.0.3282.186)
  (Driver info: chromedriver=2.36.540470 (e522d04694c7ebea4ba8821272dbef4f9b818c91),platform=Windows NT 10.0.16299 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 0 milliseconds
Build info: version: '3.11.0', revision: 'e59cfb3', time: '2018-03-11T20:26:55.152Z'
System info: host: 'DESKTOP-IGA56E0', ip: '192.168.100.233', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_91'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, acceptSslCerts: false, applicationCacheEnabled: false, browserConnectionEnabled: false, browserName: chrome, chrome: {chromedriverVersion: 2.36.540470 (e522d04694c7eb..., userDataDir: C:\Users\HePing\AppData\Loc...}, cssSelectorsEnabled: true, databaseEnabled: false, handlesAlerts: true, hasTouchScreen: false, javascriptEnabled: true, locationContextEnabled: true, mobileEmulationEnabled: false, nativeEvents: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: XP, platformName: XP, rotatable: false, setWindowRect: true, takesHeapSnapshot: true, takesScreenshot: true, unexpectedAlertBehaviour: , unhandledPromptBehavior: , version: 64.0.3282.186, webStorageEnabled: true}
Session ID: 7f077e432760af637bbc29581bc42eba
  sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
  sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
  sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
  java.lang.reflect.Constructor.newInstance(Constructor.java:423)
  org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:214)
  org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:166)
  org.openqa.selenium.remote.http.JsonHttpResponseCodec.reconstructValue(JsonHttpResponseCodec.java:40)
  org.openqa.selenium.remote.http.AbstractHttpResponseCodec.decode(AbstractHttpResponseCodec.java:80)
  org.openqa.selenium.remote.http.AbstractHttpResponseCodec.decode(AbstractHttpResponseCodec.java:44)
  org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
  org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
  org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:545)
  org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:602)
  org.openqa.selenium.remote.RemoteWebDriver.getWindowHandles(RemoteWebDriver.java:453)
  com.easy.automation.util.SeleniumUtil.switchToNextWindow(SeleniumUtil.java:38)
  com.easy.automation.testcase.TC001.test2(TC001.java:22)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1137)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
  org.testng.TestRunner.privateRun(TestRunner.java:753)
  org.testng.TestRunner.run(TestRunner.java:607)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:368)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:363)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:321)
  org.testng.SuiteRunner.run(SuiteRunner.java:270)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1284)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1209)
  org.testng.TestNG.runSuites(TestNG.java:1124)
  org.testng.TestNG.run(TestNG.java:1096)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:282)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:83)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:114)
  org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:286)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:240)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
      </failure>

          </testcase>
  
  
  
</testsuite>
